Arrays and Cycles - 2.js

```diff
@@ -0,0 +1,12 @@
+/*
+Дан массив производителей автомобилей, преобразовать массив в строку и обратно в массив
+ */
+
+let brand = ['Toyota', 'MB', 'VW', 'Porsche'];
+console.log(brand);
+
+let str = brand.toString();
```

Comment:
Tip: or you can use `join()` method.

https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/join